bin/remove.js

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+
+require('dotenv').config();
+const client = require('twilio')(process.env.ACCOUNT_SID, process.env.AUTH_TOKEN);
+const { cli } = require('cli-ux');
+
+(async () => {
+  try {
+    cli.action.start('removing');
+    const services = await client.serverless.services.list();
+    const app = services.find((service) => service.friendlyName.includes("twilio-live-interactive-audio"));
+    if (app) {
+      await client.serverless.services(app.sid).remove();
```

Comment:
Here I think we should log something just to confirm that something was deleted:

```
    if (app) {
      await client.serverless.services(app.sid).remove();
      console.log('Removed app: ' + app.domainBase);
    }
```
This will log: `Removed app: twilio-live-interactive-audio-1234`